src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -57,7 +59,7 @@ static PSVersionInfo()
             s_psVersionTable = new PSVersionHashTable(StringComparer.OrdinalIgnoreCase);
 
             s_psVersionTable[PSVersionInfo.PSVersionName] = s_psV6Version;
-            s_psVersionTable["PSEdition"] = PSEditionValue;
+            s_psVersionTable[PSVersionInfo.PSEditionName] = PSEditionValue;
             s_psVersionTable["BuildVersion"] = GetBuildVersion();
             s_psVersionTable["GitCommitId"] = GetCommitInfo();
             s_psVersionTable["PSCompatibleVersions"] = new Version[] { s_psV1Version, s_psV2Version, s_psV3Version, s_psV4Version, s_psV5Version, s_psV51Version, s_psV6Version };
```

Comment:
I did not want to distract the unrelated changes. But I agree to bring it into line if you don't mind.